h2o-core/src/main/java/water/parser/ParseSetup.java

```diff
@@ -27,6 +28,8 @@
   public static final int HAS_HEADER = 1;
   public static final int GUESS_COL_CNT = -1;
 
+  public BytesStats bytesStats;
```

Comment:
It has a value independent from ARFF; it's like wc. And it's already in master, by the way.